core/src/main/java/io/smallrye/openapi/api/OpenApiConfigImpl.java

```diff
@@ -408,6 +410,30 @@ public void doAllowNakedPathParameter() {
         return defaultConsumes;
     }
 
+    @Override
+    public Set<String> getScanProfiles() {
+        if (scanProfiles == null) {
+            String classes = getStringConfigValue(OpenApiConstants.SCAN_PROFILES);
+            if (classes == null) {
+                classes = getStringConfigValue(OpenApiConstants.SCAN_PROFILES);
+            }
```

Comment:
Not required, but if you re-base again you might rename this variable and the equivalent in the "exclude" method.